### ISSUE TITLE
Set Switch for Parameter Published Value without Subscription

### ISSUE THREAD

**Issue by stepjah** [RHEAGROUP/COMET-BatchEditor-Community-Edition#19]
### Prerequisites

- [x] I have written a descriptive issue title
- [x] I have verified that I am running the latest version of the COMET-BatchEditor
- [x] I have searched [open](https://github.com/RHEAGROUP/COMET-BatchEditor-Community-Edition/issues) and [closed](https://github.com/RHEAGROUP/COMET-BatchEditor-Community-Edition/issues?q=is%3Aissue+is%3Aclosed) issues to ensure it has not already been reported

### Description
Currently the action "SetSubscriptionSwitch" would allow to change the Switch only for Subscribed paramerers. It should also be possible to set the State of ParameterValues for EDs without a Subscription to control the PublishedValue by a dedicated Command.

### System Configuration

- [] COMET-BatchEditor version:1.1.0
- Environment (Operating system, version and so on):Win64
- .NET Framework version:
- Additional information:

**Comment by samatrhea** [comment]
Dear @stepjah ,

Are we correct in understanding you would like a new batch command that is something like: "SetParameterSwitch"?

**Comment by stepjah** [comment]
@samgerene yes, that is correct. Our use case would be to e.g. adapt the default switch in model templates faster through such a bulk operation
